puppet/zulip/files/nagios_plugins/zulip_app_frontend/check_send_receive_time

```diff
@@ -166,10 +165,16 @@ msg_to_send = str(random.getrandbits(64))
 time_start = datetime.datetime.now()
 
 if options.websocket:
-    client = WebsocketClient(host_url=options.site, sockjs_url='/sockjs/366/v8nw22qe/websocket',
-                             run_on_start=send_message_via_websocket, sender_email=sender.email,
-                             recepient_email=recipient.email, content=msg_to_send,
-                             validate_ssl=not options.insecure)
+    try:
+        sender_res = zulip_sender.register(event_types=["message"])
+        if 'error' in sender_res.get('result'):
+            report("CRITICAL", "Error subscribing to Zulips: %s" % (sender_res['msg']))
+    except Exception:
+        report("CRITICAL", "Error subscribing to Zulips:\n%s" % (traceback.format_exc()))
```

Comment:
@brainwane  I'm not sure I've taken this text from errors text for an event register above.